### PR TITLE
chore(renovate): enable automerge

### PR DESCRIPTION
**Pull request by erezrokah**
Enables renovate automerge for this repo.

I've added the PR labeler so the required label status check will pass for renovate PRs.
I've also requested to install https://github.com/apps/renovate-approve so it approves renovate PRs

### CODE REVIEW

**Review by JGAntunes — APPROVED** [review]

File: renovate.json5 [hunk 1]
```diff
@@ -3,7 +3,7 @@
   ignorePresets: [':prHourlyLimit2'],
   semanticCommits: true,
   dependencyDashboard: true,
-  automerge: false,
```
Comment (JGAntunes) [inline renovate.json5:6]: @erezrokah should we also add this to - https://github.com/netlify/node-template/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/renovate.json5 - or are we going to do it on a separate PR?

Comment (erezrokah) [reply renovate.json5:6]: Good point - I was thinking of only enabling it for this repo and note for the template itself.
I'll add it for the template too.
